lib/json.js

```diff
@@ -6,6 +6,7 @@
 
 const raw = require('raw-body');
 const inflate = require('inflation');
+const { safeParse } = require('@poppinss/utils');
```

Comment:
please can you publish a npm module with the safeParser util only `@poppinss/utils.safeparse`.

Reply:
also, I think custom json parser option here is more flexible.